app/src/main/java/bg/zahov/app/data/local/RealmManager.kt

```diff
@@ -18,49 +11,74 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.withContext
 import java.io.File
 
+/**
+ * Manages the Realm database instance, providing methods for accessing and manipulating user settings,
+ * workout states, and associated entities.
+ *
+ * This singleton class initializes and configures the database schema, handles database operations,
+ * and provides utilities for settings and workout management.
+ *
+ * @property realmConfig Lazy-loaded [RealmConfiguration] for the database schema and settings.
+ */
 class RealmManager {
+
     companion object {
-        const val LANGUAGE_SETTING = "Language"
-        const val UNIT_SETTING = "Units"
-        const val SOUND_EFFECTS_SETTING = "Sound effect"
-        const val THEME_SETTING = "Theme"
-        const val REST_TIMER_SETTING = "Rest timer"
-        const val VIBRATION_SETTING = "Vibrate upon finish"
-        const val SOUND_SETTING = "Sound"
-        const val UPDATE_TEMPLATE_SETTING = "Show update template"
-        const val FIT_SETTING = "Use samsung watch during workout"
-        const val AUTOMATIC_SYNC_SETTING = "Automatic between device sync"
+        // Constants representing various settings keys.
+        const val LANGUAGE_SETTING = "Language Settings"
```

Comment:
This can also be an enum

Reply:
I am also using some of them as text for textfields

Reply:
You can group declare them like so:
```
enum class SettingsTitles(val key: String) {
    UNIT_SETTING("Units Settings"),
    SOUND_EFFECTS_SETTING("soundEffectsSettings")
   ...
}
```

And use the strings as `it.key`, etc. 

Separately, if some of them need to be localized they can point to a string resource instead.

Reply:
Changed it to enum